include/fastrtps/log/Log.h

```diff
@@ -155,6 +155,7 @@ class Log
             std::mutex mCvMutex;
             bool mLogging;
             bool mWork;
+            volatile int current_loop;
```

Comment:
As we discussed, volatile keyword is not necessary here.

Reply:
Was this finally removed? I see it is still here?